packages/create-mc-app/src/tasks/update-application-constants.ts

```diff
@@ -33,11 +33,11 @@ function replaceEntryPointUriPathInConstants(
     retainLines: true,
   });
   if (result?.code) {
-    const prettierConfig = prettier.resolveConfig.sync(
```

Comment:
All this function are now [async only](https://prettier.io/blog/2023/07/05/3.0.0.html#change-public-apis-to-asynchronous-12574httpsgithubcomprettierprettierpull12574-12788httpsgithubcomprettierprettierpull12788-12790httpsgithubcomprettierprettierpull12790-13265httpsgithubcomprettierprettierpull13265-by-fiskerhttpsgithubcomfisker).